tests/Temporalio.Tests/Worker/WorkflowWorkerTests.cs

```diff
@@ -3873,14 +3939,28 @@ await foreach (var evt in handle.FetchHistoryEventsAsync())
         Func<TemporalWorker, Task> action,
         TemporalWorkerOptions? options = null,
         IWorkerClient? client = null)
+    {
+        await ExecuteWorkerAsyncReturning<TWf, bool>(
+            (w) =>
+            {
+                return action(w).ContinueWith(t => true);
+            },
```

Comment:
```suggestion
            w => action(w).ContinueWith(t => true),
```

But it's test code, so I don't really care